testData/solutions/win32.json5

```diff
@@ -5807,6 +6472,458 @@
+                            "description": "Changes the way mouse scrolling is passed to applications",
+                            "type": "string",
+                            "enum": ["RoutingFocus", "RoutingHybrid"]
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
+                    "ScrollFocusRoutingConfig": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "defaultInput": {
+                                "transform": {
+                                    "type": "fluid.transforms.value",
+                                    "inputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.ScrollRouting"
+                                }
+                            },
+                            "defaultOutputPath": "value",
+                            "match": {
+                                "RoutingFocus": 0,
+                                "RoutingHybrid": 1
+                            },
+                            "noMatch": 0
+                        },
+                        "path": {
+                            "transform": {
+                                "type": "fluid.transforms.literalValue",
+                                "input": "pvParam"
+                            }
+                        }
+                    }
+                },
+                "inverseCapabilitiesTransformations": {
+                    "transform": [
+                        {
+                            "type": "fluid.transforms.valueMapper",
+                            "defaultInputPath": "ScrollFocusRoutingConfig.value",
+                            "defaultOutputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.ScrollRouting",
+                            "match": {
+                                "0": "RoutingFocus",
+                                "1": "RoutingHybrid"
+                            },
+                            "noMatch": "RoutingFocus"
+                        }
+                    ]
+                }
+            },
+            "configureMouseCursorShadow": {
+                "type": "gpii.windows.spiSettingsHandler",
+                "liveness": "live",
+                "options": {
+                    "getAction": "SPI_GETCURSORSHADOW",
+                    "setAction": "SPI_SETCURSORSHADOW",
+                    "uiParam": 0,
+                    "pvParam": {
+                        "type": "BOOL"
+                    },
+                    "fWinIni": "SPIF_UPDATEINIFILE|SPIF_SENDCHANGE",
+                    "verifySettings": true
+                },
+                "supportedSettings": {
+                    "MouseCursorShadowConfig": {
+                        "schema": {
+                            "title": "Enable disable mouse cursor shadow",
```

Comment:
Confusing name - just "Enable" will do I think

Reply:
Okay, will change it, it looks simpler.